ci/tests/puppeteer/scenarios/oidc-login-strapi/init.sh

```diff
@@ -18,23 +19,24 @@ cp ${PWD}/ci/tests/puppeteer/scenarios/${SCENARIO}/strapi-custom/server.js $STRA
 
 # install node modules for strapi app
 cd $STRAPI_FOLDER/$PROJECT
-yarn install
+npm install
 
 STRAPI_CMD=./node_modules/.bin/strapi
 
 # run without SSL verification in order to call CAS via https and not worry about SSL trust
-yarn build --no-optimization
+npm run build --no-optimization
 
 # This could probably be done in javascript as part of user-permissions plugin extension to modify
 # the bootstrap config before it goes in database, but doing this until that process better documented
 echo Dumping strapi configuration
 $STRAPI_CMD configuration:dump --file config.json --pretty
-sed -i 's|\\"cas\\":{\\"enabled\\":false|\\"cas\\":{\\"enabled\\":true|g' config.json
-sed -Ei 's|cas(.*)\\"secret\\":\\"\\"|cas\1\\"secret\\":\\"strapisecret\\"|g' config.json
-sed -Ei 's|cas(.*)\\"key\\":\\"\\"|cas\1\\"key\\":\\"strapi\\"|g' config.json
-sed -i 's|subdomain\\":\\"my.subdomain.com/cas|subdomain\\":\\"localhost:8443/cas|g' config.json
+# sed on mac requires -i to specify backup file
+sed -i'.bak' 's|\\"cas\\":{\\"enabled\\":false|\\"cas\\":{\\"enabled\\":true|g' config.json
+sed -i'.bak' -E 's|cas(.*)\\"secret\\":\\"\\"|cas\1\\"secret\\":\\"strapisecret\\"|g' config.json
+sed -i'.bak' -E 's|cas(.*)\\"key\\":\\"\\"|cas\1\\"key\\":\\"strapi\\"|g' config.json
+sed -i'.bak' 's|subdomain\\":\\"my.subdomain.com/cas|subdomain\\":\\"localhost:8443/cas|g' config.json
 # this next line is fixing a bug where default redirect url contains // instead of /
-sed -i 's|1337//api|1337/api|g' config.json
+sed -i'.bak' 's|1337//api|1337/api|g' config.json
 echo Restoring modified strapi configuration
 $STRAPI_CMD configuration:restore --file config.json
 
```

Comment:
Inline sed on mac requires backup extension. Yarn was having flakiness downloading stuff and npx seemed to work better.